### PR TITLE
Refactoring meshok-react-hooks

### PR DESCRIPTION
**Pull request by codewylle**

### CODE REVIEW

File: packages/meshok-react-hooks/src/App.tsx [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import React from "react";
 import "./App.css";
-import { TodoItem } from "./components/TodoItem";
+import { TodoTree } from "./components/TodoItem";
 import { rootTodoItemData } from "meshok-e2e";
 
 export function App(): JSX.Element {
-	return <TodoItem itemData={rootTodoItemData} />;
+	return <TodoTree initData={rootTodoItemData} />;
+	// return <TodoItem itemData={rootTodoItemData} />;
```
Comment (finico) [inline packages/meshok-react-hooks/src/App.tsx]: Why did you leave it Man?

File: packages/meshok-react-hooks/src/components/TodoItem.tsx [hunk 2]
```diff
@@ -1,128 +1,125 @@
...
+
+function renderTitleText(
+	title: string,
+	isEditing: boolean,
+): JSX.Element {
+	return isEditing
+		?	<label>
+				<input type="text" defaultValue={title} />
+			</label>
+		: <label>{title}</label>;
```
Comment (finico) [inline packages/meshok-react-hooks/src/components/TodoItem.tsx]: Is it a space? It's really heavy bag to discern spaces and tabs

File: packages/meshok-react-hooks/src/components/TodoItem.tsx [hunk 3]
```diff
@@ -1,128 +1,125 @@
...
+		(n: number) =>
+			(setItemData: (prevData: ITodo) => ITodo) => {
+				setData(prevData => ({
+					...prevData,
+					children: prevData.children && prevData.children.map(
+						(value, index) =>
+							index === n ? setItemData(value) : value
+					)
+				}))
+		}, []);
```
Comment (finico) [inline packages/meshok-react-hooks/src/components/TodoItem.tsx]: Oh my bag! That was heavy to understand 🏋️‍♂️

**Comment by codewylle** [comment]
Check, man. I tried to make it lighter

**Review by finico — APPROVED** [review]

**Pull request merged**